packages/shared/flows.yaml

```diff
@@ -2189,10 +2206,10 @@ integrations:
         firstname: string
         lastname: string
         headline: string
-        account:
+        account: &ref_0
```

Comment:
where does this magic come from?

Reply:
Looks like an underlying issue with the `__extends` logic when dealing with nested objects which should be fixed ASAP if this is happening

Reply:
Not blocking this PR, but it could negatively impact the API Reference display -- would have to check